superflore/generators/bitbake/gen_packages.py

```diff
@@ -56,7 +56,7 @@ def info(string):
 def make_dir(dirname):
     try:
         os.makedirs(dirname)
-    except:
+    except Exception:
```

Comment:
I'd recommend actually checking the errno since already present is valid but if you're out of disk space etc it should fail: https://stackoverflow.com/questions/600268/mkdir-p-functionality-in-python

Reply:
Since I'm centralizing this function in #76, I'm just going to fix that there (26f99eb1571800f85da2ba428289e8e59cc12531).

Reply:
+1